pypeit/spectrographs/keck_lris.py

```diff
@@ -178,14 +178,13 @@ def compound_meta(self, headarr, meta_key):
             return binning
         elif 'lampstat' in meta_key:
             idx = int(meta_key[-2:])
-            try:
-                curr_date = time.Time(self.get_meta_value(headarr, 'mjd'), format='mjd')
-            except:
-                msgs.warn('No mjd in header. You either have bad headers, '
-                          'or incorrectly specified the wrong spectrograph, '
-                          'or are reading in other files from your directory.  '
-                          'Using 2022-01-01 as the date for parsing lamp info from headers')
-                curr_date =  time.Time("2022-01-01", format='isot')
+            curr_date = time.Time(self.get_meta_value(headarr, 'mjd'), format='mjd')
```

Comment:
This will crash when headers are bad, which was indeed the whole point of my fix. pypeit_setup was crashing. Until you have demonstrated that this does not crash on a file with bad headers, I don't agree with this change.